web/packages/teleport/src/Discover/Desktop/ConnectTeleport/RunConfigureScriptAnimation.tsx

```diff
@@ -32,7 +31,9 @@ import { JoinToken } from 'teleport/services/joinToken';
 
 const lines = (joinToken: JoinToken) => [
   {
-    text: generateCommand(cfg.getConfigureADUrl(joinToken.id)),
+    text: generateCommand(
+      'https://teleport.example/v1/webapi/scripts/desktop-access/configure/your-token/configure-ad.ps1'
```

Comment:
make it more apparent:
```suggestion
      'https://teleport.example/v1/webapi/scripts/desktop-access/configure/<YOUR_TOKEN>/configure-ad.ps1'
```

Reply:
Should we use `teleport.example.com` which is more in line with our docs than `teleport.example`?

Reply:
> though, can't we assume the length of a valid token instead?

I feel like using a realistic looking token in an example dummy URL doesn't serve any purpose and might just lead to potential confusion.